pkg/sql/compile/types.go

```diff
@@ -185,3 +187,33 @@ type RemoteReceivRegInfo struct {
 	Uuid     uuid.UUID
 	FromAddr string
 }
+
+type mergeReader struct {
```

Comment:
The code for this logic should not be placed here in the pipeline compile

Reply:
Okay, I will modify this issue